app/assets/stylesheets/houston/dashboards/staging.scss

```diff
@@ -1,31 +1,92 @@
-.on-staging-pr {
-  padding: 10px;
-  a {
-    display: inline;
-    padding-left: 10px;
-    font-size: 1.33em;
-    font-weight: 200;
-  }
-}
+#on-staging {
+  table-layout: fixed;
+  width: 100%;
+  tr.on-staging-pr {
+    padding: 10px;
 
-.staging-pr-project {
-  text-align: right;
-}
+    a {
+      display: inline;
+      padding-left: 10px;
+      font-size: 1.33em;
+      font-weight: 200;
+    }
 
-.up-next-pr {
-  padding: 10px;
-  a {
-    display: inline;
-    padding-left: 10px;
-    font-size: 1.33em;
-    font-weight: 200;
-  }
-}
+    td {
+      padding: 1px 1em 1px 0;
+      white-space: nowrap;
+
+      &.staging-pr-project{
+        text-align: right;
+        width: 75px;
+      }
+
+      &.staging-pr-title{
+        width: 99%;
+      }
+
+      &.checkboxes {
+        width: 40px;
+        text-align: right;
+      }
+      &.staging-user {
+        width: 26px;
+      }
 
-.up-next-project {
-  text-align: right;
+      .pr-tag {
+        display: inline-block;
+        width: 8px;
+        height: 8px;
+        border-radius: 4px;
+        line-height: 1;
+        margin-left: 8px;
+        &.test-needed {
+          background-color: #f7c6c7;
+        }
```

Comment:
We can assume that a PR has **test-needed** on it, so we don't need a color for that.